sympy/geometry/tests/test_polygon.py

```diff
@@ -243,8 +256,10 @@ def test_polygon():
     assert t1.medial == Triangle(Point(2.5, 0), Point(0, 2.5), Point(2.5, 2.5))
 
     # Nine-point circle
-    assert t1.nine_point_circle == Circle(Point(2.5, 0), Point(0, 2.5), Point(2.5, 2.5))
-    assert t1.nine_point_circle == Circle(Point(0, 0), Point(0, 2.5), Point(2.5, 2.5))
+    assert t1.nine_point_circle == Circle(Point(2.5, 0), 
```

Comment:
Can be worth checking these lines as well.

Reply:
In fact, I was trying to limite the line to 79 characters as the code style guide says. That's why there are this kind of problem everywherer. I was doing it in a wrong way I think. Sohuld I add a \ in the end or just roll it back to the original version?
Thank you for your advise!

Reply:
I should have run the quality test before commiting. Sorry for this mistake. I will update the code asap.